x-pack/qa/smoke-test-plugins-ssl/build.gradle

```diff
@@ -161,9 +163,11 @@ integTestCluster.dependsOn(importClientCertificateInNodeKeyStore, importNodeCert
 
 ext.pluginsCount = 0
 project.rootProject.subprojects.findAll { it.path.startsWith(':plugins:') }.each { subproj ->
-  // need to get a non-decorated project object, so must re-lookup the project by path
-  integTestCluster.plugin(subproj.path)
-  pluginsCount += 1
+  if (subproj.plugins.hasPlugin(PluginBuildPlugin) || subproj.plugins.hasPlugin(MetaPluginBuildPlugin)) {
```

Comment:
I wonder if this is too much magic. An alternative might be to write `project.project(':plugins').getChildProjects.each { n, subproj -> ... }`
to just range over direct children of `plugins`.

Reply:
Thanks. This is indeed a good suggestion. I'll update this.